pkgs/applications/editors/rstudio/remove-quarto-from-generator.patch

```diff
@@ -0,0 +1,29 @@
+--- a/src/cpp/session/CMakeLists.txt
++++ b/src/cpp/session/CMakeLists.txt
+@@ -45,8 +45,6 @@
+ 
+    if(EXISTS "${RSTUDIO_TOOLS_ROOT}/quarto")
+       set(RSTUDIO_DEPENDENCIES_QUARTO_DIR "${RSTUDIO_TOOLS_ROOT}/quarto")
+-   else()
+-      set(RSTUDIO_DEPENDENCIES_QUARTO_DIR "${RSTUDIO_DEPENDENCIES_DIR}/quarto")
+    endif()
+ 
+ endif()
+@@ -66,15 +64,7 @@
+ 
+ # validate our dependencies exist
+ foreach(VAR RSTUDIO_DEPENDENCIES_DICTIONARIES_DIR
+-            RSTUDIO_DEPENDENCIES_MATHJAX_DIR
+-            RSTUDIO_DEPENDENCIES_PANDOC_DIR
```

Comment:
This is a bit more than just quarto. Just to understand, you're removing this so we can use `nixpkgs` mathjax & pandoc?

Reply:
Ok, can you reflect that in the patch filename so noone thinks it's by accident?

Reply:
See answer in https://github.com/NixOS/nixpkgs/pull/185154#discussion_r973501024